docker-compose.localapp.yml

```diff
@@ -0,0 +1,57 @@
+version: '3'
+services:
```

Comment:
Services names from this file conflict with services names from `docker-compose.yml`. I think you have to add a prefix for all resources from this file.